src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx

```diff
@@ -60,3 +60,9 @@ If you’ve accidentally sent sensitive data to the server, you likely don't to
 - If you send the data as a tagged value, removing the event is not enough. Visit your **Project Settings > Tags** to permanently remove any related data for a given tag.
 - While you cannot delete a single event, you can delete the issue the event is in by clicking the trash icon on the **Issue Details** page.
 - If a large number of unique events - representing multiple issues - contain the data, you may need to delete and re-create the project to effectively cleanse the system.
+
+## Known Limitations of server-side data scrubbing
+
+The following limitations generally apply to all server-side data scrubbing.
+
+- It is currently not possible to avoid scrubbing data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) by adding the category name added to Safe Fields. Only the message and data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) are used for data scrubbing. And is custom JSON data is provided the fields will be scrubbed according to the configured rules and Safe Fields.
```

Comment:
```suggestion
- It is currently not possible to avoid scrubbing data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) by adding the category name added to Safe Fields. Only the message and data in [_breadcrumbs_](/product/issues/issue-details/breadcrumbs/) are used for data scrubbing. And if custom JSON data is provided the fields will be scrubbed according to the configured rules and Safe Fields.
```